docs/en/enterprise-edition/policy-reference/aws-policies/aws-iam-policies/bc-aws-355.adoc

```diff
@@ -25,9 +25,7 @@
 
 === Description
 
-This policy is ensuring that no IAM (Identity and Access Management) policies on AWS (Amazon Web Services) allow the use of "*" as a statement's resource for restrictable actions. Allowing "*" can be dangerous as it means "all", including all possible services and resources, which can lead to a much wider range of access than intended. This can potentially expose sensitive information and actions, allowing for unauthorized manipulation of resources. By restricting the use of "*", we reduce the chance of giving an actor more permissions than they need, a concept known as the Principle of Least Privilege (PoLP), and thus minimize potential damage.
-
-For more info, visit https://cloudsplaining.readthedocs.io/en/latest/glossary/resource-exposure/[cloudsplaning documentation].
+This policy is checking to ensure that no IAM (Identity and Access Management) policy documents allow "\*" as a statement's resource for actions that can and should be restricted. Prisma Cloud checks the AWS documentation for IAM actions that can be restricted to a resource and recommends defining a specific resource rather than "\*". If "\*", which stands for all resources, is used, it means the statement grants permissions to all AWS resources which is potentially unsafe. This is poor practice because it broadens the scope of access controls and increases the risk of unauthorized access. It's important to define granular permissions to each user access, as unrestricted access can lead to unwanted manipulations or data breaches. Therefore, it's recommended to specify restrictions and assign minimum necessary access rights.
```

Comment:
This policy checks IAM policies for statements that allow unrestricted resource access ('*') for actions that can and should be restricted to specific resources.  This behavior is potentially unsafe because it broadens the scope of access controls and increases the risk of unauthorized access.
Prisma Cloud checks the AWS documentation for IAM actions that can be restricted to a resource and recommends defining a specific resource rather than '\*'.  For example, the s3:PutObject action can be restricted to a specific S3 bucket instead of allowing uploads to any S3 bucket using '\*'. 
It is best security practice to define granular permissions to each user access, as unrestricted access can lead to unwanted manipulations or data breaches. Therefore, it is recommended to specify restrictions and assign minimum necessary access rights.